rasa/core/actions/action.py

```diff
@@ -780,7 +784,31 @@ async def run(
             return []
 
 
+class ActionUnlikelyIntent(Action):
+    """An action that indicates that the intent predicted by NLU is unexpected.
+
+    This action can be predicted by `IntentTEDPolicy`.
+    """
+
+    def name(self) -> Text:
+        """Returns the name of the action."""
+        return ACTION_UNLIKELY_INTENT_NAME
+
+    async def run(
+        self,
+        output_channel: "OutputChannel",
+        nlg: "NaturalLanguageGenerator",
+        tracker: "DialogueStateTracker",
+        domain: "Domain",
+    ) -> List[Event]:
+        """Runs action. Please see parent class for the full docstring."""
+        evts = await super().run(output_channel, nlg, tracker, domain)
```

Comment:
I'm confused by the use of `super()` here: doesn't this call the parent call's run method which raises `NotImplementedError`?

Reply:
In fact, you're correct here — I've just updated it to return `[]`